config/locales/en/errors.yml

```diff
@@ -98,8 +98,8 @@ en:
         prior_authority/steps/ufn_form:
           attributes:
             ufn:
-              blank: Enter your unique file number for this application
-              invalid: UFN must be a 9-digit number
+              blank: Enter the unique file number
+              invalid: Unique file number must be in the correct format, for example 310224/001
```

Comment:
this one i also find odd. We have already have a hint on the field "For example 310224/001" so that seems redundant, making it unwieldily long.